matminer/datasets/tests/test_convenience_loaders.py

```diff
@@ -0,0 +1,181 @@
+import pandas as pd
+import unittest
+
+from matminer.datasets.tests.base import DataSetTest
+from matminer.datasets.convenience_loaders import load_glass_ternary_hipt, \
```

Comment:
In sklearn and keras, the loaders for the datasets are in the dataset module. Example, 

```python
from sklearn.datasets import load_iris
iris = load_iris
```

The pedantic side of me wants to suggest to overhaul our library to follow this convention. But, the only practical benefit of moving things from `convenience_loaders` into `datasets` are the names being quicker to type. So, I am just pointing this semantic difference out without any intention of making a stink about if you disagree

Reply:
It's a tough call, because the way seaborn does things is with the load_dataset(string_identifier) method like so.

```python
from seaborn import load_dataset
iris = load_dataset("iris")
```

Initially we thought this method would make for a better interface but then I realized it would be nice to have dataset specific loading options so I added the convenience loaders. But then you get this problem of which one do you consider the "right" way of getting datasets. I like keeping things where they are with the convenience loaders in their own namespace to show they aren't considered the "primary interface" but rather utilities for user convenience.

Reply:
Keeping them separate sounds like a good plan to me. Thanks for walking me through your thinking about this!